game/test_endpoints.py

```
@@ -0,0 +1,260 @@
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(response.url, reverse('verify_otp'))
+        self.assertFalse(mock_user.is_active)
+        mock_user.save.assert_called()
+        mock_send_mail.assert_called_once()
+        self.assertIn('registration_user_id', self.client.session)
+        user_id = self.client.session['registration_user_id']
+        self.assertEqual(user_id, mock_user.id)
+
+    @mock.patch('game.views.send_mail', side_effect=Exception('SMTP err'))
+    @mock.patch('game.views.CustomUserCreationForm')
+    def test_register_post_email_failure(self, mock_form_class, mock_mail):
```

Ruff (0.15.18)

[warning] Unused method argument: `mock_mail`

(ARG002)